vault/resource_aws_auth_backend_client_test.go

```diff
@@ -117,6 +117,27 @@ func TestAccAWSAuthBackendClientStsRegionNoEndpoint(t *testing.T) {
 	})
 }
 
+func TestAccAWSAuthBackendClientStsRegionFromClient(t *testing.T) {
+	backend := acctest.RandomWithPrefix("aws")
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion115)
+		},
+		Providers:    testProviders,
+		CheckDestroy: testAccCheckAWSAuthBackendClientDestroy,
+		Steps: []resource.TestStep{
+			{
```

Comment:
Would be nice to add an initial step to test the default behavior and also ensuring that importing the field works 😄